CHANGES.rst

```diff
@@ -367,6 +369,8 @@ New Features
 
 - ``astropy.vo``
 
+  - KeyError when converting Table v1.2 numeric arrays fixed. [#4781]
```

Comment:
I did. Changeset bfc403ccf4fa0a684b48cf3d6b5b16e57c5c0029

On Thu, 2016-04-14 at 13:30 -0700, P. L. Lim wrote:
> In CHANGES.rst:
> 
> > @@ -367,6 +369,8 @@ New Features
> >  
> >  - ``astropy.vo``
> >  
> > +  - KeyError when converting Table v1.2 numeric arrays fixed. [#4781]
> 
> Remove duplicate change log entry.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> 
> 

-- 
Al Niessner

I have never found the companion that was so companionable as solitude.
- From Walden by Henry David Thoreau

The universe is indifferent, and life is brutal; however, it is man's
choice of behavior that makes them malevolent rather than benevolent.

Some will fall in love with life and drink it from a fountain
That is pouring like an avalanche coming down the mountain.
- From the song Pepper by the Butthole Surfers